cpp/oneapi/dal/backend/serialization.hpp

```diff
@@ -187,7 +187,7 @@ class serialization_ids {
     ID(6040000000, decision_forest_model_interop_impl_reg_id);
 
     // Algorithms - Linear Regression
-    ID(7010000000, linear_regression_norm_eq_model_impl_id);
+    ID(7010000000, linear_regression_model_impl_id);
```

Comment:
See the question about serialization ID above.

Reply:
We need to save this ID for the sake of backward compatibility between models. Naming is changed according to a new class name

Reply:
Discussed